recipe/conda_build_config.yaml

```diff
@@ -18,10 +18,8 @@ m2w64_c_stdlib:                # [win]
   - m2w64-sysroot              # [win]
 m2w64_c_stdlib_version:        # [win]
   - 12                         # [win]
-c_stdlib_version:              # [unix]
-  - 2.17                       # [linux]
-  - 10.13                      # [osx and x86_64]
-  - 11.0                       # [osx and arm64]
+c_stdlib_version:   # [osx and x86_64]
+  - 10.14           # [osx and x86_64]
```

Comment:
Ah, we did break stuff here though. And like for https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/5559, we're in the situation where we need to keep the migration open to not bump the macos baseline, see https://github.com/conda-forge/conda-forge.github.io/issues/2467

Reply:
Fixed in 3a222220ffab33116c5601592322e0cd67352493